symfony-cmf/routing-bundle/2.0/config/packages/cmf_routing_bundle.yaml

```diff
@@ -0,0 +1,15 @@
+# Read the documentation: https://symfony.com/doc/master/cmf/bundles/routing/configuration.html
+# Install phpcr stack before using this configuration
+#cmf_routing:
+#    chain:
+#        routers_by_id:
+#            cmf_routing.dynamic_router: 200
+#            router.default:             100
+#    dynamic:
+#        persistence:
+#            phpcr:
+#                enabled: true
+
+framework:
+    templating: { engines: ['twig'] }
```

Comment:
Unfortunately, we cannot accept bundles that depend on the Templating component. It will be deprecated at some point, as all core bundles have migrated to use Twig directly. So, if the bundle cannot work with Twig directly, I'd recommend to create a new version.

Reply:
Found that only one line needs `templating` https://github.com/symfony-cmf/routing-bundle/blob/47be1b0eb96e5ab7db5fa921a0342e38f3915795/src/Validator/Constraints/RouteDefaultsValidator.php#L48  i think we should find a replacement somehow.

Reply:
make your validator accept either Twig or Templating as argument, so that it works when the templating system is not used.